foojay-resolver/src/test/kotlin/org/gradle/toolchains/foojay/FoojayApiTest.kt

```diff
@@ -48,6 +48,10 @@ class FoojayApiTest {
           Arguments.of(16, any(), true, OperatingSystem.LINUX, Architecture.X86_64),
           Arguments.of(8, GRAAL_VM, false, OperatingSystem.WINDOWS, Architecture.X86_64),
           Arguments.of(20, GRAAL_VM, false, OperatingSystem.LINUX, Architecture.X86_64),
+          Arguments.of(22, any(), false, OperatingSystem.AIX, Architecture.PPC64),
+          Arguments.of(22, any(), false, OperatingSystem.LINUX, Architecture.PPC64LE),
+          Arguments.of(22, any(), false, OperatingSystem.LINUX, Architecture.S390X),
```

Comment:
What is the reason/logic behind these combinations? If so, could you also add `Arguments.of(11, any(), false, OperatingSystem.LINUX, Architecture.S390X),`, because 22 is very modern, but temurin eg does not provide a jdk for Java 8 for Linux on s390x but only for JDK 11.

Reply:
No particular reasoning, just testing if the parameter translation works or not.